include/ibamr/IBFESurfaceMethod.h

```diff
@@ -300,6 +309,16 @@ class IBFESurfaceMethod : public IBStrategy
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::CoarsenSchedule<NDIM> > >& u_synch_scheds,
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::RefineSchedule<NDIM> > >& u_ghost_fill_scheds,
         double data_time);
+    /*!
+     * Compute the exterior fluid traction for when using the
```

Comment:
Should we make it easy to do either interior or exterior tractions?

Reply:
I actually used to only compute the exterior traction but depends on which side the interesting stuff is happening either the interior or exterior calculation becomes important. Also at least for the sake of convergence study it turns out that numerically du/dn^- is not exactly equal to -([[du/dn]] - du/dn^+) does that make sense?....however maybe we can just keep one variable and a flag to to switch the normal to determine which side needs to be calculated for a specific problem?

Reply:
I think that sometimes we will want one or the other?

I _think_ it makes sense that interpolating from one side is not exactly the same as interpolating from the other side and then adding the jump. That is why I think it would be good to be able to do either. :-)

Reply:
That's right. There were some code exploring the options for taking the mean value of the two or other ideas, but I removed them. The example I used to explore this was the falling sphere (to see which option results in smoother traction on the surface). However I think I can come up with a simpler RBD example to check this maybe, like a 2D disk which runs much faster. The main goal is to have both smoother (help with stability) and more accurate traction values.